inc/patterns/general-pricing-table.php

```diff
@@ -11,12 +11,12 @@
 					<hr class="wp-block-separator is-style-wide"/>
 					<!-- /wp:separator -->
 
-					<!-- wp:heading {"style":{"typography":{"fontSize":"clamp(3rem, 6vw, 4rem)","lineHeight":"0.5"}}} -->
-					<h2 id="1" style="font-size:clamp(3rem, 6vw, 4rem);line-height:0.5">' . esc_html( _x( '1', 'First item in a numbered list.', 'twentytwentytwo' ) ) . '</h2>
+					<!-- wp:heading {"style":{"typography":{"fontSize":"clamp(2.75rem, 6vw, 3.25rem)","lineHeight":"0.5"}}} -->
```

Comment:
Should these also use the custom variable with the `clamp` value as the fallback? e.g.

```suggestion
					<!-- wp:heading {"style":{"typography":{"fontSize":"var(--wp--custom--typography--font-size--gigantic, clamp(2.75rem, 6vw, 3.25rem))","lineHeight":"0.5"}}} -->
```

Reply:
Yep, they should. 👍